plugins/postcss-is-pseudo/src/split-selectors/specificity.ts

```diff
@@ -0,0 +1,128 @@
+import parser from 'postcss-selector-parser';
```

Comment:
Wondering if we should extract this to be available elsewhere but you have more idea here.

Reply:
Going to make an issue for this.
Need to figure out how we handle reusable bits of code

Reply:
I created an issue for this.